### PR TITLE
SD-1340 Improve behavior of desk definition dialog

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: apps/stages.py [hunk 1]
```diff
@@ -51,6 +56,19 @@ class StagesResource(Resource):
 
 
 class StagesService(BaseService):
+    def on_create(self, docs):
+        for doc in docs:
+            if not doc.get('desk'):
+                doc['desk_order'] = 1
+                return
```
Comment (amagdas) [inline apps/stages.py]: If `docs` has more than one item this return will break the `desk_order` for the rest if the first one doesn't have `'desk'` set.

**Comment by amagdas** [comment]
Some tests are needed for this to see that the order is properly handled.

**Comment by ioanpocol** [comment]
:+1:  (except needed tests)

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**